src/Uno.Toolkit.UI/Controls/DrawerFlyout/DrawerFlyoutPresenter.cs

```diff
@@ -125,6 +125,7 @@ T FindTemplatePart<T>(string name) where T : class =>
 					_initOnceOnLoaded = false;
 
 					_popup = FindHostPopup() ?? throw new Exception("Unable to find host popup.");
+					UpdateOpenness(false);
```

Comment:
why between two unrelated _popup lines?
move it below: _drawerContentPresenter.SizeChanged